docs/dm/zh_CN/get-started.md

```diff
@@ -47,12 +47,12 @@ Get Started
 # Master Configuration.
 
 [[deploy]]
-mysql-instance = "172.16.10.81:3306"
-dm-worker = "172.16.10.72:10081"
+mysql-instance = "172.16.10.81:3306"   // instance id
+dm-worker = "172.16.10.72:10081"       // worker id
```

Comment:
`worker address` is better? because we only set worker address in `dm-worker.toml` or `inventory.ini` in DM-ansible, no `worker id` item exists

Reply:
Auxiliary understood, is not enough to only change, cause a problem how not associated `dm - worker ID` in `dmctl`